all_seaing_driver/all_seaing_driver/rover_lora_controller.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import rclpy
+from rclpy.node import Node
+from all_seaing_interfaces.msg import ControlOption
+import serial
+import struct
+import time
+
+class RoverLoraController(Node):
+    def __init__(self):
+        super().__init__('rover_lora_controller')
+
+        # Setup serial connection
+        self.serial_port = serial.Serial('/dev/ttyUSB0', 57600, timeout=1)
+        time.sleep(2)  # Allow serial port to stabilize
+        
+        self.data_size = struct.calcsize('Bddd')
```

Comment:
What is Bddd?

Reply:
Byte double double double. This is Wells stuff tho and tbh should not be in my merge but ehhhh.

Reply:
it was needed for the test on tuesday